docs/book/features/orphan_volumes.md

```diff
@@ -0,0 +1,50 @@
+
+This functionality provides a set of APIs to detect and delete orphan volumes on-demand, and also provide an option to turn on automatic deletion periodically.
+
+### What qualifies as an orphan volume ?
+A volume qualifies as an orphan volume if it meets all of the below conditions:
+
+1) A PersistentVolume in any of the registered kubernetes clusters is not using the volume.
+2) Volume was dynamically provisioned using vSphere CSI driver.
+3) Volume exists for more than 50 minutes(or `orphan-volume-detection-interval-mins` value configured in `cnsmanager-config` configmap) at the time of orphan volumes detection.
+
+### Which orphan volumes are skipped from detection/deletion ?
+Some volumes, even if a kubernetes PersistentVolume doesn't map to them, will not be considered during orphan volume detection/deletion. These include:
+* Volumes created out of band (not using vSphere CSI driver).
+* Statically provisioned CNS volume whose name doesn't start with `pvc-`.
+* File volumes.
+* Orphan volumes that contain snapshots will be detected as orphans but they can not be deleted using Orphan volume delete API.
+
+### A word of caution!
+Orphan volumes once deleted from a datastore cannot be recovered. It is therefore imperative that you [register all the Kubernetes clusters](../../../README.md#register-kubernetes-clusters-before-you-start) in vCenter with CNS Manager, so that an orphan volume is detected correctly and no crucial data loss happens when they're deleted.
+
+### APIs provided
+1. *GET /orphanvolumes* 
+
+This API takes optional parameters, datacenter & list of datastores, and returns orphan volumes for them. 
+- If datacenter is not specified, then it returns all orphan volumes in the vCenter (all datastores on all datacenters ).
+- If only datacenter is specified, then it returns orphan volumes in all datastores in the datacenter.
+- If both datacenter & list of datastores is specified, it returns orphan volumes in specified datastores on the datacenter.
+
+Detection of orphan volumes can be a time-consuming operation if there are large number of orphans. Hence it is performed asynchronously at regular intervals and the response is cached. This API returns list of orphan volumes computed in the last run, along with the next operation interval(`RetryAfterMinutes`).  
+**Note:** For newly deployed CNS manager application when orphan volumes are being computed in the background for the first time, the API may return no orphan volumes. It should then be re-tried after `RetryAfterMinutes` to get orphan volumes computed in the latest run.
+
+
+2. *DELETE /orphanvolumes*
+
+This API is used to delete orphan volumes detected earlier. It also takes optional parameters, datacenter & list of datastores, and deletes orphan volumes from them.  
+You can also specify whether you want to delete orphan FCDs attached to a virtual machine or not. If set to `true`, the API will detach the orphan volume from the VM before deleting it.  
```

Comment:
Since FCD is an internal implementation detail, can we just say something like `You can also specify whether you want to delete orphan volumes`?